src/main/java/techcourse/myblog/custom/LoginUserResolver.java

```diff
@@ -0,0 +1,31 @@
+package techcourse.myblog.custom;
+
+import techcourse.myblog.exception.UnauthorizedException;
+import techcourse.myblog.user.Information;
+import techcourse.myblog.user.User;
+
+import org.springframework.core.MethodParameter;
+import org.springframework.web.bind.support.WebDataBinderFactory;
+import org.springframework.web.context.request.NativeWebRequest;
+import org.springframework.web.context.request.RequestAttributes;
+import org.springframework.web.method.support.HandlerMethodArgumentResolver;
+import org.springframework.web.method.support.ModelAndViewContainer;
+
+public class LoginUserResolver implements HandlerMethodArgumentResolver {
+	@Override
+	public boolean supportsParameter(MethodParameter parameter) {
+		return parameter.getParameterType().isAssignableFrom(User.class);
```

Comment:
`@LoginUser` 어노테이션은 아래와 같이 쓰입니다.

```suggestion
		return parameter.hasParameterAnnotation(LoginUser.class);
```

Reply:
수정했습니다~! 감사합니다!